### DISCUSSION TITLE
Mention throws TypeError s.domNode.getBoundingClientRect is not a function

### DISCUSSION THREAD

**Discussion by taranjeetiterate** [afconsult/quill-mention#288] [Q&A]
Trying to mention users with @ character in the quill editor throws TypeError s.domNode.getBoundingClientRect is not a function in the console and the suggestions are not shown.

Steps for Reproduction
The error is a little tricky to reproduce as only a few of my company's clients occasionally experience the issue on production.
screenshot below
<img width="1425" alt="Screenshot 2022-09-14 at 4 58 15 PM" src="https://user-images.githubusercontent.com/92028581/190190131-95fe2d6a-7a5c-417e-993e-26372df44c76.png">

Expected behavior:
The mention feature should show the suggestions

Actual behavior:
The following error is thrown in the console and the suggestions don't work

TypeError
s.domNode.getBoundingClientRect is not a function
Platforms:

browser: Chrome 104.0.5112.101 (Official Build) (arm64)
device: Mac
os: Mac OS X 11.4 (BigSur)

**Comment by kblizeck** [comment]
Still getting this error in `quill-mention@v3.4.1` - an included package in `quill`.
